src/20240108_AaveV2Polygon_ReserveFactorUpdatesJan152024/ReserveFactorUpdatesJan152024.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Reserve Factor Updates (Jan 15, 2024)"
+author: "karpatkey_TokenLogic"
+discussions: "https://governance.aave.com/t/arfc-reserve-factor-updates-polygon-aave-v2/13937"
+---
+
+## Simple Summary
+
+This AIP is a continuation of AIP-284 and increases the Reserve Factor (RF) for assets on Polygon v2 by 5.00%, up to a maximum of 99.99%.
```

Comment:
It seems reasonable for me to always put the previous proposal doing increases, instead of the first one

Reply:
done!